layers/stateless/sl_render_pass.cpp

```diff
@@ -325,6 +330,41 @@ bool Device::ValidateCreateRenderPass(VkDevice device, const VkRenderPassCreateI
                 }
             }
         }
+
+        const VkAttachmentDescriptionFlags both_skip_enable_transfer_flags =
+            VK_ATTACHMENT_DESCRIPTION_RESOLVE_SKIP_TRANSFER_FUNCTION_BIT_KHR |
+            VK_ATTACHMENT_DESCRIPTION_RESOLVE_ENABLE_TRANSFER_FUNCTION_BIT_KHR;
+        if ((attachment.flags & both_skip_enable_transfer_flags) == both_skip_enable_transfer_flags) {
+            skip |= LogError("VUID-VkAttachmentDescription-flags-11773", device, create_info_loc.dot(Field::flags), "is %s.",
+                             string_VkAttachmentDescriptionFlags(attachment.flags).c_str());
+        }
+
+        if ((attachment.flags & (VK_ATTACHMENT_DESCRIPTION_RESOLVE_SKIP_TRANSFER_FUNCTION_BIT_KHR |
+                                 VK_ATTACHMENT_DESCRIPTION_RESOLVE_ENABLE_TRANSFER_FUNCTION_BIT_KHR)) != 0) {
+            if (!enabled_features.maintenance10) {
+                skip |= LogError("VUID-VkAttachmentDescription-flags-11775", device, attachment_loc.dot(Field::flags),
+                                 "is %s but maintenance10 feature is not enabled.",
+                                 string_VkAttachmentDescriptionFlags(attachment.flags).c_str());
+            } else if (!phys_dev_ext_props.maintenance10_props.resolveSrgbFormatSupportsTransferFunctionControl) {
+                skip |= LogError("VUID-VkAttachmentDescription-flags-11774", device, attachment_loc.dot(Field::flags),
+                                 "is %s but resolveSrgbFormatSupportsTransferFunctionControl feature is not available.",
```

Comment:
```suggestion
                                 "is %s but resolveSrgbFormatSupportsTransferFunctionControl feature is not supported.",
```

wording we use everywhere else